e2e/cypress/support/ui/index.js

```diff
@@ -13,3 +13,4 @@ import './search';
 import './system';
 import './channel';
 import './team';
+import './compliance_export';
```

Comment:
Please arrange these imports alphabetically. Thanks.

Reply:
Thanks @josephbaylon for guidance 👍